handler.go

```diff
@@ -72,11 +71,13 @@ func (ca *Cagent) Run(outputFile *os.File, interrupt chan struct{}) {
 				// for error codes 5xx, wait for configured amount of time and try again
 				retries++
 				if retries > ca.Config.OnHTTP5xxRetries {
-					log.Errorf("hub connection error. giving up")
+					log.Errorf("Run: hub connection error. giving up")
 					return
 				}
-				log.Infof("hub connection error %d/%d, retrying in %v s", retries, ca.Config.OnHTTP5xxRetries, ca.Config.OnHTTP5xxRetryInterval)
+				log.Infof("Run: hub connection error %d/%d, retrying in %v s", retries, ca.Config.OnHTTP5xxRetries, ca.Config.OnHTTP5xxRetryInterval)
```

Comment:
Maybe would be better to use `Warn` instead of `Info`?

Reply:
From issue description:
> Failed connections are logged with severity info. Only if all retries have failed, an error is logged.